src/fidesops/ops/schemas/connection_configuration/connection_secrets_bigquery.py

```diff
@@ -22,6 +22,23 @@ class KeyfileCreds(BaseModel):
     auth_provider_x509_cert_url: Optional[str] = None
     client_x509_cert_url: Optional[str] = None
 
+    class Config:
+        """Add example for each field"""
+        schema_extra = {
+            "example": {
+                "type": "service_account",
+                "project_id": "XXXXXXXXXXXXXXXXXXXX",
+                "private_key_id": "key-id",
+                "private_key": "-----BEGIN PRIVATE KEY-----\nprivate-key\n-----END PRIVATE KEY-----\n",
+                "client_email": "service-account-email",
+                "client_id": "XXXXXXXXXXXXXXXXXXXX",
+                "auth_uri": "https://accounts.google.com/o/oauth2/auth",
+                "token_uri": "https://accounts.google.com/o/oauth2/token",
+                "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
+                "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/service-account-email"
```

Comment:
@pattisdr what should our convention be wrt real world URLs for connection secrets? I'd prefer we linked to the provider's docs at most, is that reasonable?

Reply:
Makes sense to me